package.json

```diff
@@ -96,13 +96,15 @@
     "loopback-context": "^1.0.0",
     "mocha": "^5.2.0",
     "nyc": "^10.1.2",
-    "phantomjs-prebuilt": "^2.1.7",
     "sinon": "^6.1.4",
     "sinon-chai": "^3.2.0",
     "strong-error-handler": "^3.0.0",
     "strong-task-emitter": "^0.0.8",
     "supertest": "^3.0.0"
   },
+  "optionalDependencies": {
```

Comment:
Please not that `optionalDependencies` are installed in production, e.g. when an app depends on loopback. With your change in place, every LoopBack application is going to install phantomjs! That would be a poor user experience IMO.

We need to mark `phantomjs-prebuilt` as an optional dev-dependency instead.

Unfortunately, `npm` does not support optional dev-dependencies out of the box, at least as far as I can tell from searching the internet and the discussion in https://github.com/npm/npm/issues/3870.

A possible workaround is offered by this npm package https://www.npmjs.com/package/optional-dev-dependency

Considering that we have a single optional dev-dependency only, it may be simpler to simply execute `npm install --no-save phantomjs-prebuilt` from our Gruntfile. (Maybe check if the package has not been already installed, to speed up repeated execution of `npm t` for framework developers practicing test-driven development.)